### PR TITLE
Incorporate review feedback from oleg-nenashev

### PR DESCRIPTION
**Pull request by alxsap**
See https://github.com/jenkinsci/parameterized-remote-trigger-plugin/pull/32#pullrequestreview-100987235

### CODE REVIEW

**Comment by marcusholl** [comment]
... and where is the incorporated review feedback? I can only see technical changes ...

**Comment by alxsap** [comment]
Those are the review comments: https://github.com/jenkinsci/parameterized-remote-trigger-plugin/pull/32#pullrequestreview-100987235
And in this PR the changes are being prepared.

File: src/main/java/org/jenkinsci/plugins/ParameterizedRemoteTrigger/RemoteJenkinsServer.java [hunk 1]
```diff
@@ -201,9 +199,9 @@ public static Auth2Descriptor getDefaultAuth2Descriptor() {
 
     @Override
     public RemoteJenkinsServer clone() throws CloneNotSupportedException {
-        RemoteJenkinsServer clone = new RemoteJenkinsServer();
+        RemoteJenkinsServer clone = (RemoteJenkinsServer)super.clone();
```
Comment (alxsap) [inline src/main/java/org/jenkinsci/plugins/ParameterizedRemoteTrigger/RemoteJenkinsServer.java]: primitive fields already set in the object returned by super.clone()

File: src/main/java/org/jenkinsci/plugins/ParameterizedRemoteTrigger/auth2/NoneAuth.java [hunk 2]
```diff
@@ -60,6 +63,7 @@ public int hashCode() {
 	}
 	
 	@Override
+	@SuppressFBWarnings("EQ_UNUSUAL")
 	public boolean equals(Object obj) {
```
Comment (alxsap) [inline src/main/java/org/jenkinsci/plugins/ParameterizedRemoteTrigger/auth2/NoneAuth.java]: single line `this.getClass().isInstance(obj)` would be enough

File: src/main/java/org/jenkinsci/plugins/ParameterizedRemoteTrigger/auth2/NullAuth.java [hunk 3]
```diff
@@ -61,6 +62,7 @@ public int hashCode() {
 	}
 	
 	@Override
+	@SuppressFBWarnings("EQ_UNUSUAL")
```
Comment (alxsap) [inline src/main/java/org/jenkinsci/plugins/ParameterizedRemoteTrigger/auth2/NullAuth.java]: same here

**Pull request merged**